test/test_pep621.py

```diff
@@ -0,0 +1,62 @@
+import unittest
+
+from flake8_requirements.checker import Flake8Checker
+from flake8_requirements.checker import ModuleSet
+from flake8_requirements.checker import memoize
+
+try:
+    from unittest import mock
+    builtins_open = 'builtins.open'
+except ImportError:
+    import mock
+    builtins_open = '__builtin__.open'
+
+
+class Pep621TestCase(unittest.TestCase):
+
+    def setUp(self):
+        memoize.mem = {}
+        self.content = """
+        [project]
+        name="test"
+        dependencies=["tools==1.0"]
+
+        [project.optional-dependencies]
+        dev = ["dev-tools==1.0"]
+        """
+
+    def test_get_pyproject_toml_pep621(self):
```

Comment:
```suggestion
class Pep621TestCase(unittest.TestCase):

    content = """
    [project]
    name = "test"
    dependencies = ["tools==1.0"]

    [project.optional-dependencies]
    dev = ["dev-tools==1.0"]
    """

    def setUp(self):
        memoize.mem = {}

    def test_get_pyproject_toml_pep621(self):
```

Reply:
No problem. I've pushed the requested changes.